frontend/playground-frontend/src/pages/feedback.tsx

```diff
@@ -70,20 +68,29 @@ const Feedback = () => {
       email.trim() &&
       feedback.trim()
     ) {
-      /*
-      setSuccessful(
-        await send_feedback_mail(firstName, lastName, email, feedback)
-      );*/
+      if (recaptcha != "") {
+        const emailResult = await sendFeedback({
+          email_address: process.env.REACT_APP_FEEDBACK_EMAIL,
+          subject: "FEEDBACK - " + firstName + " " + lastName + " " + email,
+          body_text: feedback,
+        });
+        if (!emailResult.isSuccess) {
+          toast.error(emailResult.data.message);
+        }
+        console.log(emailResult);
```

Comment:
do we need this? @Dashrekker

Reply:
it helps with confirming a successful submission but i can remove it

Reply:
my comment was referring to the console.log. i understand the need for the other pieces. @Dashrekker